.circleci/config.yml

```diff
@@ -85,12 +81,27 @@ jobs:
           name: Deploy to S3
           command: aws s3 sync workspace/target s3://$RELEASES_BUCKET_NAME/build-$CIRCLE_SHA1/ --delete
 
+  docker-deploy:
+    machine: true
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run: docker login -u $DOCKER_USER -p $DOCKER_PASS
+      # because target/ has been put into place `make image` can be
+      # ran without running ./script/package (which would require npm)
+      - run: cd ops/docker && make image
+      - run: docker push cljdoc/cljdoc
+
 workflows:
   version: 2
   build-and-deploy:
     jobs:
       - build
       - prettier
+      - docker-deploy:
+          requires:
+            - build
```

Comment:
This needs a branch filter like deploy I think?

Reply:
The used Docker tag contains the branch name if it isn't `master` but yeah, we probably just shouldn't push to Docker unless we're on `master`.